com.unity.template-universal/Packages/manifest.json

```diff
@@ -6,9 +6,7 @@
     "com.unity.analytics": "3.6.11",
     "com.unity.collab-proxy": "1.7.1",
     "com.unity.ext.nunit": "1.0.6",
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.ide.visualstudio": "2.0.9",
-    "com.unity.ide.vscode": "1.2.3",
+    "com.unity.feature.development": "1.0.1",
     "com.unity.purchasing": "3.2.2",
     "com.unity.render-pipelines.universal": "13.0.0",
     "com.unity.test-framework": "1.1.29",
```

Comment:
No need for this one, and we should remove the modules as well

Reply:
We can try, but last time we have try to remove test-framework nothing was working... (on yamato side)